lib/solid_queue/configuration.rb

```diff
@@ -36,6 +36,14 @@ def configured_processes
       end
     end
 
+    def valid?
+      skip_recurring_tasks? || recurring_tasks.none? || recurring_tasks.all?(&:valid?)
```

Comment:
I feel like this is probably not exhaustive enough, but I'm unsure of other cases to expand upon. This is probably sufficient for the scope of this PR though.

Reply:
Could we include perhaps the case of no configured processes? So we could unify that call in the Supervisor, instead of having two separate checks? 

Besides that, I think you can skip `recurring_tasks.none?` because if `recurring_tasks` is empty, `recurring_tasks.all?(&:valid?)` will return `true`.

Reply:
Ugh... seems so obvious after you pointed it out 😅 

I hope you don't mind, but I've kept the two distinct error messages. It feels like the verbose invalid task error message would be helpful.